src/docker-osx-dev

```diff
@@ -701,7 +703,7 @@ function do_rsync {
 }
 
 #
-# Usage: sync [PATHS ...]
+# Usage: do_sync [PATHS ...]
```

Comment:
Following the pattern of "do_rysnc" for the low level operation, leaving "sync" for running the higher level sync when called through the CLI